vault/activity_log_test.go

```diff
@@ -2313,6 +2313,12 @@ func TestActivityLog_Deletion(t *testing.T) {
 	core, _, _ := TestCoreUnsealed(t)
 	a := core.activityLog
 
+	// Wait for retention worker to finish its first pass.
+	select {
+	case <-a.retentionDone:
```

Comment:
do you want to have a timeout in this `select` so that it'll fail gracefully if `retentionDone` isn't closed for some reason?

Reply:
Yeah, that's probably best. I thought about it, figured it was unlikely, but I should have gone with my first reaction.